config/auth.js

```diff
@@ -0,0 +1,34 @@
+/*eslint no-undef: "error"*/
```

Comment:
isn't it your default behaviour already, defined in .eslintrc?

Reply:
this was removed